DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/staticIndexes/projectIndexResults/metadata.json

```diff
@@ -0,0 +1,215 @@
+                "  Used in step 7.",
+                "",
+                "* Note: it is not mandatory for this class to be nested within the index class."
+            ],
+            "Assets": [
+            ]
+        },
+        {
+            "Title": "The index constructor",
+            "Slug": "step-4",
+            "DescriptionLines": [
+                "* The _Map function_ defines the index-fields content - which fields from the 'Employee' document will compose each index-field.",
+                "",
+                "* The index built will contain a list of indexed terms for each index-field specified in the Map function, and a mapping to the relevant document.",
+                "",
+                "* These indexed terms can later be queried on to get their matching documents, the documents that contain them.",
+                "",
+                "* In this example, the 'HiredAt.Year' data is indexed in the _WorkingInCompanySince_ index-field."
+            ],
+            "Assets": [
+                {
+                    "Title": "Map Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/java/indexes/map-indexes"
+                },
+                {
+                    "Title": "Static Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/10-static-indexes-and-other-advanced-options#static-indexes-and-other-advanced-options"
+                },
+                {
+                    "Title": "Working with Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/12-working-with-indexes#working-with-indexes"
+                }
+            ]
+        },
+        {
+            "Title": "Query for documents",
+            "Slug" : "step-5",
+            "DescriptionLines": [
+                "* Query the 'Employees' collection using the defined index.",
+                "",
+                "* The 'Employees' documents can be _filtered_ by the index-fields defined in `Employees_ByWorkPeriod.IndexEntry`",
+                "",
+                "* The resulting documents will include only employees that started working after 'startYear'."
+            ],
+            "Assets": [
+                
+                {
+                    "Title": "Filtering Results - Basics Demo",
+                    "Type" : "demo",
+                    "Url"  : "/java/queries/filtering-results-basics"
+                }
+            ]
+        },
+        {
+            "Title": "Project Results",
+            "Slug" : "step-6",
+            "DescriptionLines": [
+                "* `selectFields()` must be used to satisfy the compiler and be able to select fields from the Employee class type,",
+                "  otherwise the compiler will only allow access to the IndexEntry class details.",
```

Comment:
This comment was added in the C# demo regarding the `.OfType` usage.
but here you are referring to `selectFields()` ....

Is it relevant here ?
Shouldn't you add `ofType` to the java demo ? similar to the C# demo ?

Reply:
What does it means?
Should I use ofType in the code instead selectFields()? 
Because it is not look like both can do exactly the  same things in java.

Reply:
we don't need ofType when we use select fields.

Reply:
@Drorlev 
Pls check this bullet - is it needed at all ?

Reply:
Done

Reply:
I removed that first bullet